delegatedSending.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Delegated Sending
+ * @module delegatedSending
+ */
+
+const internal = require('./internal/delegatedSending');
+
+/**
+ * Return the amount of invites left in the pool
+ * @param {Number} pool - Pool number
+ * @return {Promise<Number>} Number of invites remaining
+ */
+module.exports.pools = internal.pools
+module.exports.invitesInPool = internal.pools
```

Comment:
What's the reason for having duplicate functions here (and below)?

Reply:
While `pools()` matches the contract's function name, it isn't exactly informative. Offering an alias here, `invitesInPool()`, should lead to more readable code down the line.